### PR TITLE
Latest lichess-bot PRs

### PR DESCRIPTION
**Pull request by TheYoBots**

### CODE REVIEW

**Comment by OIVAS7572** [comment]
@TheYoBots 
for what is this pull request

**Comment by TheYoBots** [comment]
It’s for pushing all useful lichess-bot pull requests here. I will complete it soon.

**Comment by OIVAS7572** [comment]
Ok
I asked, just for curiosity

**Comment by TheYoBots** [comment]
@OIVAS7572 this pull request is almost done, I will merge it soon. Just wanted to mention something about automatic deploys so that users know what it does. Do you have any suggestions on what to write about automatic deploys?

**Comment by TheYoBots** [comment]
@OIVAS7572 tell me if I should do anything more here.

**Comment by OIVAS7572** [comment]
> @OIVAS7572 this pull request is almost done, I will merge it soon. Just wanted to mention something about automatic deploys so that users know what it does. Do you have any suggestions on what to write about automatic deploys?

no need of mentioning automatic deploy

File: README.md [hunk 1]
```diff
@@ -14,14 +14,15 @@
 **Do note that in certain operating systems Heroku CLI doesn't get added to path automatically. If that's the case you'll have to add heroku to your path manually.**
 - Run this command in cmd or powershell `heroku stack:set container -a appname`, where `appname` is replaced with your Heroku app's name.
 - In heroku, in the `Deploy` tab click on `Connect to GitHub` and then click on `search` and select your fork/import of this repository.
-- Now scroll down and under `Manual deploy`, click on `deploy` with the `master` branch selected.
+- Now scroll down and under `Manual deploy`, click on `deploy` with the `master` branch selected. <br/>
+**Do note that you could also `Enable Automatic Deploys` with the `master` branch selected if you would like each commit you make to get automatically and easily deployed onto your bot. It is your choice whether you'd like to Enable or Disable Automatic Deploys.**
```
Comment (TheYoBots) [inline README.md:18]: I’ve mentioned it’s optional @OIVAS7572

Comment (OIVAS7572) [reply README.md:18]: if it is optional no need to wright it as bold
I suggest to put it in a brackets

**Review by OIVAS7572 — APPROVED** [review]

**Pull request merged**